publish-swagger-docs.sh

```diff
@@ -8,6 +8,7 @@ then
   exit 0
 fi
 
+./gradlew clean installDist
```

Comment:
shouldn't this be done in the travis script section?
https://github.com/hmcts/draft-store/blob/master/.travis.yml#L24

Reply:
imho, it feel more 'self contained' when it's here